app/stac_api/views.py

```diff
@@ -31,6 +32,17 @@
 logger = logging.getLogger(__name__)
 
 
+def debug_print_request_data(view):
+    if sys.getsizeof(view.request.data) < 500000:
+        logger.debug(
+            "Request: %s, request.data: %s",
+            view.request,
+            view.request.data,
+        )
```

Comment:
The logging should make uses of the extra and uses `info` like in logging middleware; `logger.info('Request %s %s', method, path, extra={'request': request})`) the `data` should then be added to the logging configuration 
```yaml
filters:
  django:
    (): logging_utilities.filters.django_request.JsonDjangoRequest
    include_keys:
      - request.path
      - request.method
      - request.headers
      - request.data
    exclude_keys:
      - request.headers.Authorization
      - request.headers.Proxy-Authorization
      - request.headers.Cookie
``` 
NOTE: one could think that simply changing the log configuration would be enough due to our logging middleware, however in the logging middleware the request object is a django HttpRequest object that don't have a `.data`. It only has a `.body` which is a bytestring.